FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -166,23 +166,39 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </objectDesc>
                          
                         <bindingDesc>
-                            <binding contemporary="partly"></binding>
+                            <binding contemporary="partly">
                             <decoNote xml:id="b1" type="Boards">Two wooden boards.</decoNote>    
                             <decoNote xml:id="b2" type="SewingStations">4</decoNote>
                             <decoNote xml:id="b3" type="bindingMaterial"><material key="wood"></material></decoNote>
+                            </binding>
```

Comment:
it is only here that the binding description is over and the `binding `tag must be closed here

Reply:
All these things have already been corrected, please update your local version by clicking PULL in your GitHub Desktop

For an overview of the corrections which I had already carried out to get a valid XML please carefully study https://github.com/BetaMasaheft/Manuscripts/pull/1978/commits/5782128e5a0d55b56c66615ac017ae2a2f90f456

just read my comments and pay attention to the corrections already carried out and highlighted there